packages/driver/cypress/fixtures/click-event-by-type.html

```diff
@@ -1,19 +1,19 @@
 <!DOCTYPE html>
```

Comment:
With your previous change, was this implemented? https://github.com/cypress-io/cypress/issues/7318

Reply:
I read those 2 issues. And below are my opinions:

### #7318

I think #19541 and #7318 are separate issues. #7318 wants to listen to `search` event with `cy.type('{enter}')`. But the problem is that [`search` event is not a standard event](https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement/search_event) as @jennifer-shehane wrote in the issue. 

I guess we can ignore this in this PR.


### #8267

Yes and no. 

There are 2 problems in #8267.

1. `cy.get('#button').focus().type('{enter}')` should trigger `click` event. => It's solved by #19726. 
2. `cy.get('#button').type('f')` should not send click event. => It's not solved and the debate isn't ended. 

As in [the comment of #8767](https://github.com/cypress-io/cypress/issues/8267#issuecomment-743918524), we should decide the current behavior of `cy.type` first. 

It assumes that `click` is required before typing into an element. But as we all know, "click" isn't the only way to make elements get focus. 

I believe #8267 should be handled after the decision about this has been made.